tests/tasks/setup_ipa.yml

```diff
@@ -10,9 +10,9 @@
   when: __is_beaker_env
 
 - name: Clone ansible-freeipa repo
+  # noqa git-latest
   git:
     repo: https://github.com/freeipa/ansible-freeipa
-    version: master
```

Comment:
Here's what I meant about the use of 'master'. We can safely pin version 'v1.9.0' and update it if we need in the future.

On the maintainance cost of updating it, I believe it will be updated soon, but not often.

Reply:
Why can't we just omit `version` altogether, then git will use whatever is the default branch?

Reply:
If we can, it would be better. I though it was required.

Reply:
It is not required by the `git module`, but it is by `ansible-lint`. So, we are adding this line to skip the check by `ansible-lint`.
https://github.com/linux-system-roles/certificate/pull/142/files#diff-d1ca26ec3f441ccd0c4bb18b601d19668a8ffacfab0c71694fea7b46adc9823cR13